db/seeds.rb

```diff
@@ -6,24 +6,32 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-18.times do |t|
-  Kitchen.create(floor: t)
+puts 'Creating kitchens 3 to 18'
+16.times do |t|
```

Comment:
We could make the code more descriptive with ruby `Range`. This way the comment is not necessary. 

```suggestion
(3..18).each do |t|
  Kitchen.create(floor: t)
end  
```